File: maple/maple-user-web/src/main/webapp/js/service/resumeService.js
//服务层
app.service('resumeService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../resume/findAll.do');		
	}
    this.findOnebyuserid=function(){
        return $http.get('../resume/findOneByuserid.do');
    }
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../resume/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../resume/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../resume/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../resume/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../resume/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../resume/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: maple/maple-type-interface/src/main/java/com/maple/service/ResumeService.java
package com.maple.service;
import java.util.List;
import com.maple.pojo.Resume;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ResumeService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<Resume> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(Resume resume);
	
	
	/**
	 * 修改
	 */
	public void update(Resume resume);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public Resume findOne(int id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(int[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(Resume resume, int pageNum, int pageSize,String name);

	public Resume findOneByuserid(String userid);
}
